src/components/NewAppModal.js

```diff
@@ -125,20 +134,15 @@ export const NewAppModal = _.flow(
             ]),
             li({ style: { marginTop: '1rem' } }, [
               'Cloud Compute size of ', span({ style: { fontWeight: 600 } },
-                [`${getMachineDetails().cpu} CPUS, ${getMachineDetails().memory}  GB of memory, 50 GB disk space`])
-              //TODO: Define the disk space using DEFAULT_DISK_SIZE from the mob_pd branch
+                // Temporarily hard-coded disk size, once it can be customized this should be revisited
+                [`${cpu} CPUS, ${memory} GB of memory, 50 GB disk space`])
```

Comment:
I think this disk size may be wrong. I think we're creating with 30GB disks? Testing as soon as I have a workspace that isn't messed up.

Reply:
Kai tested on this branch, this should be 30 GB disk space.